harness/isConstructor.js

```diff
@@ -8,6 +8,10 @@ defines: [isConstructor]
 ---*/
 
 function isConstructor(f) {
+    if (typeof f !== "function") {
+      throw new Test262Error("isConstructor invoked with a non-function vale");
```

Comment:
```suggestion
      throw new Test262Error("isConstructor invoked with a non-function value");
```

Reply:
It's a shame, "vale" is such a cool word.